packages/cactus-common/src/test/typescript/unit/objects/get-all-field-names.ts

```diff
@@ -0,0 +1,30 @@
+import test, { Test } from "tape";
```

Comment:
@gabrielsoaresm94 I think this one can be deleted as well. It has the same content as `packages/cactus-common/src/test/typescript/unit/objects/get-all-field-names.test.ts`
Maybe another case of the rebase/merge gone bad? I renamed this file in my commit to have it contain a `.test` suffix (it's a pattern we intend to roll out for the entire codebase eventually)

Reply:
@petermetz I did not merge automatically, because of that some files remained, but I finished correcting it.